src/Lean/Meta/Tactic/Simp/Main.lean

```diff
@@ -201,7 +201,7 @@ inductive SimpLetCase where
   | nondepDepVar -- `let x := v; b` is equivalent to `(fun x => b) v`, but result type depends on `x`
   | nondep -- `let x := v; b` is equivalent to `(fun x => b) v`, and result type does not depend on `x`
 
-def getSimpLetCase (n : Name) (t : Expr) (v : Expr) (b : Expr) : MetaM SimpLetCase := do
+def getSimpLetCase (n : Name) (t : Expr) (_ : Expr) (b : Expr) : MetaM SimpLetCase := do
```

Comment:
```suggestion
def getSimpLetCase (n : Name) (t : Expr) (b : Expr) : MetaM SimpLetCase := do
```

This renaming seems very counter-productive.. not only does it break `getSimpLetCase (v := ...)` and remove the only information about what the argument should be (the *v*alue), but it also hides the fact that the argument is actually unused.

Reply:
A more clever script definitely would do something more sensible for parameter names